usaspending_api/reporting/v2/views/submission_history.py

```diff
@@ -13,10 +13,11 @@ class SubmissionHistory(PaginationMixin, AgencyBase):
     endpoint_doc = "usaspending_api/api_contracts/contracts/v2/reporting/agencies/toptier_code/fiscal_year/fiscal_period/submission_history.md"
 
     def get(self, request, toptier_code, fiscal_year, fiscal_period):
-        self.fiscal_year = int(fiscal_year)
+        validated_params = self.validated_url_params
+        fiscal_year = validated_params["fiscal_year"]
+        fiscal_period = validated_params["fiscal_period"]
```

Comment:
Are `fiscal_year` and `fiscal_period` still necessary in the method parameters now that they are coming from `validated_params`? I see this in a few other places in the PR

Reply:
They are not necessary, but I chose to left them since they are still coming throw the `request.kwargs`. Serves as a reminder that those are being sent over in the actual URL. `validated_url_params` just validates / fetches some of the kwargs. Similar to how we have treated `toptier_code`.